### PR TITLE
Fixing the previously PR

### PR DESCRIPTION
**Pull request by agutierrez-eb**
In this PR I fix the errors in the previous PR ( #8 ). I delete unnecessary comments and the unused imports. 
In this PR, there is the new script. The unit test are for the old script and now we add a new integration test for check both scripts throw the same result.

### CODE REVIEW

**Comment by facundochaud-eb** [comment]
nit: you can link another PR in the description by just typing `#8`: #8

File: invoicing_app/management/commands/generate_tax_receipts_new.py [hunk 1]
```diff
@@ -4,16 +4,18 @@
...
 from datetime import datetime as dt
 from dateutil.relativedelta import relativedelta
 
 from invoicing import settings
 
 from invoicing_app.models import PaymentOptions, Event, Order
-from memory_profiler import profile
-from django.db import connection
+
+# from django.db import connection
```
Comment (juanvallejo-eb) [inline invoicing_app/management/commands/generate_tax_receipts_new.py]: You can also remove these comments

Comment (agutierrez-eb) [reply invoicing_app/management/commands/generate_tax_receipts_new.py]: Ok! I'll do it

**Review by juanvallejo-eb — APPROVED** [review]
nice clean up!

**Pull request merged**